critwall/src/main/java/org/opencadc/critwall/FileSyncJob.java

```diff
@@ -282,12 +280,12 @@ private void doSync() {
                     }
                     
                     if (!success) {
-                        log.info("FileSyncJob.SLEEP " + artifactID + " dt=" + RETRY_DELAY[retryCount]);
+                        log.info("FileSyncJob.SLEEP dt=" + RETRY_DELAY[retryCount]);
                         Thread.sleep(RETRY_DELAY[retryCount++]);
                     }
                 }
                 if (!success) {
-                    msg = " retryCount=" + retryCount + " download-attempts=" + numAttempts;
+                    msg = " all URLs tried, retryCount=" + retryCount;
```

Comment:
please remove the extra cruft and use the msg that was there; if/when we convert to some structured logging or ingest into a queryable system that free text is just in the way.

Reply:
Removed msg on failure.